<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimatix</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>210</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>210</height>
       </size>
      </property>
      <property name="title">
       <string>ParameterSet</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>531</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>爬坡时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="upDownTime">
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>稳定时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="holdTime">
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>电压：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Vol">
            <property name="value">
             <number>85</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>30</y>
         <width>521</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>编码器分辨率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>X方向DPI：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>起始喷孔：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QDoubleSpinBox" name="encodeSpinBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>9999.989999999999782</double>
            </property>
            <property name="singleStep">
             <double>0.001000000000000</double>
            </property>
            <property name="value">
             <double>0.586000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="xDpi">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="starNol">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>喷头类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Y方向DPI：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>终止喷孔：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="noTypecomboBox">
            <item>
             <property name="text">
              <string>256</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="yDpi">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="endNol">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>256</number>
            </property>
            <property name="value">
             <number>256</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>Info</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>需要打印</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bigPass">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>大Pass</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="smallPass">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>小Pass</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>文件地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>D:/result_V2.bmp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>选择...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>闪喷开启</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>闪喷关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="status_lab">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.617978 rgba(21, 52, 255, 255), stop:1 rgba(255, 255, 255, 255));border-radius: 15px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="startbtn">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>开始打印</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="endBtn">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>停止打印</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="x_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="xPos">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="x_dcc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="x_start">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>x_s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="x_end">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>x_e</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="yPos">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>U:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="uPos">
        <property name="text">
         <string>u</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Connected:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="connectStatus">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
